src/Packages/Microsoft.Extensions.StaticAnalysis/build/config/Benchmark-Tier1.globalconfig

```diff
@@ -1,8 +1,8 @@
 # Created by DiagConfig, the diagnostic config generator
-# Generated : 2023-05-24 21:09:55Z
+# Generated : 2023-08-05 20:41:44Z
 # Max Tier  : 1
-# Attributes: general, general_externalonly, performance
-# Analyzers : ILLink.RoslynAnalyzer, Microsoft.AspNetCore.App.Analyzers, Microsoft.AspNetCore.Components.Analyzers, Microsoft.CodeAnalysis.CSharp.NetAnalyzers, Microsoft.CodeAnalysis.NetAnalyzers, Microsoft.Extensions.ExtraAnalyzers.Roslyn4.0, Microsoft.Extensions.LocalAnalyzers, Microsoft.VisualStudio.Threading.Analyzers, Microsoft.VisualStudio.Threading.Analyzers.CSharp, SonarAnalyzer.CSharp
+# Attributes: general, performance
+# Analyzers : ILLink.RoslynAnalyzer, Microsoft.Analyzers.Extra, Microsoft.Analyzers.Local, Microsoft.AspNetCore.App.Analyzers, Microsoft.AspNetCore.Components.Analyzers, Microsoft.CodeAnalysis.CSharp.NetAnalyzers, Microsoft.CodeAnalysis.NetAnalyzers, Microsoft.VisualStudio.Threading.Analyzers, Microsoft.VisualStudio.Threading.Analyzers.CSharp, SonarAnalyzer.CSharp
```

Comment:
What is our breaking change stance in terms of changing these files from stable version to stable version? If we consider that we shouldn't do it, it would be good to separately setup a quick review with some engineers in the compiler team to make sure our opinions here seem correct.

Reply:
The changes in here, which all come from the files in eng/Diags, usually fall into two categories:

* New warnings being added to an analyzer
* Warnings being turned off because they are found to be redundant. This happens a lot with the Sonar analyzers in particular.